pkg/plugins/policies/meshhttproute/plugin/v1alpha1/listeners.go

```diff
@@ -29,7 +29,7 @@ func generateFromService(
 	proxy *core_xds.Proxy,
 	clusterCache map[common_api.BackendRefHash]string,
 	servicesAcc envoy_common.ServicesAccumulator,
-	rules rules.Rules,
```

Comment:
I'm not sure if we can do that. We'll break the configuration for old `kuma.io/service` tag service no?

Reply:
I just pass more generic object, and then take either old toRules or new ResourceRules